cypress/integration/learn/donate/donate-page-default.js

```diff
@@ -0,0 +1,60 @@
+/* global cy */
+
+const selectors = {
+  donateSupport: {
+    firstTitle: '.donate-support h4:first-of-type b',
+    secondTitle: '.donate-support h4:last-of-type b',
+    firstText: '.donate-support p:first-of-type',
+    secondText: '.donate-support p:last-of-type',
+    link: '.donate-support a'
+  }
+};
+
+describe('Donate page', () => {
+  before(() => {
+    cy.clearCookies();
+    cy.login();
```

Comment:
```suggestion
    cy.exec('npm run seed');
    cy.login();
```

Pretty minor, but the last test was failing for me here since the developmentuser was already a donor. This will reset them in the db before running the test.